### DISCUSSION TITLE
Opschalingsanalyse vraag 2: Wat betekenen deze veranderingen voor de gemeentelijk organisatie?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#252] [Pilot Gemeente]
VNG Opschalingsanalyse vraag 2: **Wat betekenen deze veranderingen voor de gemeentelijk organisatie?** Met deze veranderingen wordt gedoeld op vraag 1 van de VNG Opschalingsanalyse #251

**Comment by WOostdam** [comment]
- Aanpassing van bestaande processen zodat registratie daar onderdeel van wordt.
- Eigenaar bepalen
- Kwaliteitsmedewerker bepalen
- Beheerder(s) bepalen
- Inkoop- aankoopbeleid aanpassen

**Reply by marcvanandel** [reply]
Zijn dit zaken die 'eigenlijk' al bepaald hadden moeten worden en nu zichtbaar(der) worden?

**Reply by mikegroels** [reply]
'Eigenlijk' wel ja. Bij ons was het al zichtbaar, maar er is nu extra aandacht voor.

**Comment by marcvanandel** [comment]
(zie ook antwoorden / comments op #251 )

**Comment by marcvanandel** [comment]
Dit zou wel eens een kernregistratie kunnen worden (zelfde niveau als basisregistratie) wat betekent dat financiering en aandacht ook vergelijkbaar ingeregeld moeten worden - niet iets 'erbij doen' dus 😉 

Oproep: Geef het de aandacht die het verdient!
